lib/src/msg.c

```diff
@@ -266,6 +268,9 @@ cc_msg_t* cc_msg_builder(int device_id, int command, const void *data_struct)
         // list count
         *pdata++ = assignment->list_count;
 
+        //list bitmask
```

Comment:
this is going to break backwards compatibility in a nasty way, because old versions do not have this data.
we cannot change the data structures in non-backwards-compatible way.
the same applies to cc-slave.
(plugging in old non-v0.4.0 footswitch would crash the host, because older cc-slave passes the `list_count` here)

I have an idea for a backwards-compatible way to handle this..